### PR TITLE
docs: add aliases

### PR DESCRIPTION
**Pull request by steverfrancis**

### CODE REVIEW

File: content/docs/how-to-guides/SAML-and-omni/how-to-add-saml-users/index.md [hunk 1]
```diff
@@ -3,24 +3,23 @@ title: "Add a User to Omni with SAML Enabled"
 description: "A guide on how to add a user to Omni with SAML authentication enabled."
 draft: false
 weight: 60
+aliases:
+  - ../how-to-add-saml-users/
```
[the remark this answers is not in the document]
Comment (steverfrancis) [reply content/docs/how-to-guides/SAML-and-omni/how-to-add-saml-users/index.md:7]: They seem to work at least on the deploy preview...

File: content/docs/how-to-guides/how-to-create-a-cluster/index.md [hunk 2]
```diff
@@ -57,6 +65,20 @@ install:
...
+kind: Cluster
+name: example
+kubernetes:
+  version: v1.27.0
+talos:
+  version: v1.5.2
+features:
+  diskEncryption: true
+  ```
+
```
Comment (TimJones) [inline content/docs/how-to-guides/how-to-create-a-cluster/index.md:79]: Indentation of closing code fence is off
~~~suggestion
features:
  diskEncryption: true
```
~~~

File: content/docs/how-to-guides/SAML-and-omni/how-to-add-saml-users/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TimJones) [reply content/docs/how-to-guides/SAML-and-omni/how-to-add-saml-users/index.md:7]: Yup, this instance is properly configured already.

**Review by TimJones — APPROVED** [review]

**Pull request merged**